ubuntu/ubuntu-20.x/ubuntu-20.04-hpc/install.sh

```diff
@@ -23,7 +29,10 @@ $UBUNTU_COMMON_DIR/install_lustre_client.sh
 # install mellanox ofed
 ./install_mellanoxofed.sh
 
-# install nvidia gpu driver  
+# install mpi libraries
```

Comment:
Any particular reason for moving the MPI install earlier?

Reply:
Consistency with Alma mostly.